client/dogs/FormPage.js

```diff
@@ -243,82 +273,106 @@ export default class IndexPage extends React.Component {
-              <label htmlFor="lookup-dogAge" className="txt-l">
+              <label htmlFor="lookup-dogYear" className="txt-l">
                 Dog Age (Years)
               </label>
               <input
-                id="lookup-dogAge"
-                name="dogAge"
-                value={this.state.dogAge}
+                id="lookup-dogYear"
+                type="text"
+                name="dogYear"
+                value={this.state.dogYear}
                 onChange={this.handleInputChange}
                 placeholder="2, 0.5, 17..."
                 className="txt-f"
               />
             </div>
 
-            <div className="sel">
-              <label htmlFor="lookup-breed" className="sel-l">
-                Dog Breed
+            <div className="txt">
+              <label htmlFor="lookup-dogMo" className="txt-l">
+                Dog Age (Months)
               </label>
-              <div className="sel-c">
-                <select
-                  name="breed"
-                  id="lookup-breed"
-                  value={this.state.breed}
-                  className="sel-f">
-                  <option value="Alaskan Malamute">Alaskan Malamute</option>
-                  <option value="Alaskan Husky">Alaskan Husky</option>
-                  <option value="Pit Bull Terrier">Pit Bull Terrier</option>
-                  <option value="Bulldog">Bulldog</option>
-                  <option value="Australian Shephard">
-                    Australian Shephard
-                  </option>
-                  <option value="Basset Hound">Basset Hound</option>
-                  <option value="Italian Greyhound">Italian Greyhound</option>
-                  <option value="Otterhound">Otterhound</option>
-                  <option value="Pharoah Hound">Pharoah Hound</option>
-                  <option value="Plott Hound">Plott Hound</option>
-                  <option value="Portuguese Water Dog">
-                    Portuguese Water Dog
-                  </option>
-                  <option value="Whippet">Whippet</option>
-                </select>
+              <input
+                id="lookup-dogMo"
+                type="text"
+                name="dogMo"
+                value={this.state.dogMo}
+                onChange={this.handleInputChange}
+                placeholder="2, 0.5, 17..."
+                className="txt-f"
+              />
+            </div>
+
+            <div className="fs-c m-b300">
+              <div className="sel">
+                <label htmlFor="lookup-breed" className="sel-l sel-l--mt000">
+                  Primary Breed
+                </label>
+                <div className="sel-c sel-c--fw">
+                  <select
+                    name="breed"
+                    id="lookup-breed"
+                    value={this.state.breed}
+                    className="sel-f">
+                    <option value="Alaskan Malamute">Alaskan Malamute</option>
```

Comment:
Might be good to have an array of strings of breed names somewhere, and build this list of `<option>`s by `map`ping over that list.